src/Containers.jsx

```diff
@@ -65,10 +173,10 @@ class Containers extends React.Component {
         let columns = [
             { name: container.Name, header: true },
             image,
-            container.Config.Cmd.join(" "),
+            container.Config.Cmd ? container.Config.Cmd.join(" ") : "",
```

Comment:
`null` and `undefined` should be perfectly valid for showing in a table. Why is that necessary?

Reply:
If inspectContainer return a container without Cmd attribute it can not use join.

Reply:
Right, I just meant the `""` part. But nevermind, it's just a tiny nitpick, it's fine as-is.